src/components/admin/UserStatusDropdown.tsx

```diff
@@ -49,7 +47,7 @@ export const UserStatusDropdown: React.FC<UserStatusDropdownProps> = ({
   );
 
   return (
-    <div className={classNames("UserStatusDropdown", containerClassName)}>
+    <div className={classNames("UserStatusDropdown flex", containerClassName)}>
```

Comment:
Same here for the `.scss` file classNames.

Reply:
as agreed, moved class name info to `data-bem`:  https://github.com/sparkletown/sparkle/pull/2931/commits/c90bfc174c869b913ac93ae9776b886e624279c8